lte/cloud/go/services/lte/obsidian/models/defaults.go

```diff
@@ -38,7 +38,8 @@ func NewDefaultTDDNetworkConfig() *NetworkCellularConfigs {
 			GxGyRelayEnabled:         swag.Bool(false),
 			CloudSubscriberdbEnabled: false,
 			CongestionControlEnabled: swag.Bool(true),
-			EnableConvergedCore:      swag.Bool(false),
+			Enable5gFeatures:         swag.Bool(false),
+			NodeIdentifier:           "192.168.200.1",
```

Comment:
why don't you leave it as blank string `""`

Reply:
Yes, we can put blank here. But, It requires IP address. So, I put it here. I have tested with blank entry also. If its fine with empty string I will update that.